python/tests/core/test_base_table_scan.py

```diff
@@ -14,11 +14,12 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
 from iceberg.api import Schema
 from iceberg.api.types import IntegerType, NestedField
+import pytest
 
 
+@pytest.mark.parametrize("ts_table", ['one', 'none'], indirect=True)
```

Comment:
Is this still needed with the `ts_table` fixture already parametrized?

Reply:
thanks for that. You are correct! fixed.